analyzer/src/funTest/assets/projects/synthetic/spdx-project-expected-output.yml

```diff
@@ -79,11 +79,11 @@ packages:
       algorithm: ""
   vcs:
     type: "Git"
-    url: "<REPLACE_URL>"
-    revision: "<REPLACE_REVISION>"
-    path: "analyzer/src/funTest/assets/projects/synthetic/spdx/project/libs/openssl"
+    url: "ssh://github.com/openssl/openssl.git"
```

Comment:
Was it intended that the placeholders are replaced by actual values? (I assume, this is the case, but want to make sure.)

Reply:
Yes, that's intended. The VCS info now comes from the download location string, not from ORT's own Git repo, so there is no need anymore to use the placeholders.